### DISCUSSION TITLE
We had the feedback from DDDEurope, let's have a retro!

### DISCUSSION THREAD

**Discussion by philou** [murex/mikado-testbuilders-kata#57] [General]
Hi all,

Please share if you prefer:

1. to run an 'async' retro through this tread?
2. or to meet all together for 1h?

If you want to meet, what are your availabilities?

Thanks,

Philippe

cc @mengdaming @aatwi @cadichris @pdalesme @AntoineMx

**Comment by mengdaming** [comment]
I prefer option 2

**Comment by pdalesme** [comment]
Option 2 also for me

Le mar. 9 mars 2021 à 17:18, Damien Menanteau ***@***.***> a
écrit :

**Comment by AntoineMx** [comment]
Same here

**Comment by aatwi** [comment]
second options as well

**Comment by CadiChris** [comment]
Hi ! Sorry I'm late, option #2 for me too. Wednesday at 5.30 PM would be great.

**Comment by philou** [comment]
We are all in for option2. Unfortunately, I am not available on Wednesdays at 5:30. Any other day would be fine at 5:30 PM. What are your availabilities?

**Reply by CadiChris** [reply]
Thursday, 5:40 would work for me !

**Reply by philou** [reply]
Great! @cadichris can you prepare a blank Miro board for all of us? It looks like it's easier for you.

**Reply by CadiChris** [reply]
Miro board ! https://miro.com/app/board/o9J_lOm8sDY=/
